content-warehouse/src/main/java/contentwarehouse/v1/UpdateDocumentSchema.java

```diff
@@ -0,0 +1,104 @@
+import java.util.concurrent.TimeoutException;
+
+public class UpdateDocumentSchema {
+  public static void updateDocumentSchema() throws IOException, 
+        InterruptedException, ExecutionException, TimeoutException { 
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "your-project-id";
+    String location = "your-region"; // Format is "us" or "eu".
+    String documentSchemaId = "your-document-schema-id";
+    updateDocumentSchema(projectId, location, documentSchemaId);
+  }
+
+  public static void updateDocumentSchema(String projectId, String location, 
+        String documentSchemaId) throws IOException, InterruptedException,
+          ExecutionException, TimeoutException { 
+    String projectNumber = getProjectNumber(projectId);
+
+    String endpoint = "contentwarehouse.googleapis.com:443";
+    if (!"us".equals(location)) {
+      endpoint = String.format("%s-%s", location, endpoint);
+    }
+
+    DocumentSchemaServiceSettings documentSchemaServiceSettings = 
+             DocumentSchemaServiceSettings.newBuilder().setEndpoint(endpoint).build(); 
+
+    // Create the Schema Service Client 
+    try (DocumentSchemaServiceClient documentSchemaServiceClient = 
+            DocumentSchemaServiceClient.create(documentSchemaServiceSettings)) {
+            
+      //The full resource name of the location, e.g.: 
+      //projects/{project_number}/location/{location}/documentSchemas/{document_schema_id}
+      DocumentSchemaName documentSchemaName = 
+          DocumentSchemaName.of(projectNumber, location, documentSchemaId);
+            
+      // Define the new Schema Property with updated values
+      PropertyDefinition propertyDefinition = PropertyDefinition.newBuilder()
+          .setName("stock_symbol")
+          .setDisplayName("Searchable text")
+          .setIsSearchable(true)
+          .setIsRepeatable(true)
+          .setIsRequired(false)
+          .setTextTypeOptions(TextTypeOptions.newBuilder().build()).build();
+
+      DocumentSchema updatedDocumentSchema = DocumentSchema.newBuilder()
+                    .setDisplayName("My Test Schema") 
+                    .addPropertyDefinitions(0, propertyDefinition).build();
+
+      // Create the Request to Update the Document Schema
+      UpdateDocumentSchemaRequest updateDocumentSchemaRequest = 
+            UpdateDocumentSchemaRequest.newBuilder()
+            .setName(documentSchemaName.toString())
+            .setDocumentSchema(updatedDocumentSchema).build();
+            
+      // Update Document Schema
+      updatedDocumentSchema = 
+        documentSchemaServiceClient.updateDocumentSchema(updateDocumentSchemaRequest);
+            
+      // Read the output of Updated Document Schema Name
+      System.out.println(updatedDocumentSchema.getName());
+    }
+  }
+
+  private static String getProjectNumber(String projectId) throws IOException { 
+    try (ProjectsClient projectsClient = ProjectsClient.create()) { 
```

Comment:
Add the client initializer comment.

`// Initialize client that will be used to send requests. This client only needs to be created once, and can be reused for multiple requests.`